### ISSUE TITLE
Add field for license on spring initializer web page

### ISSUE THREAD

**Issue by CodingSpiderFox** [spring-io/initializr#727]
As a developer, I want to be able to choose an open source license (e.g. GPLv3, Apache 2.0, EPL, etc.) when creating a project online with spring initializer.

According to the chose license, a suitable license file should be added to the downloaded zip

**Comment by snicoll** [comment]
@CodingSpiderFox I don't understand the use case. This is about getting started with an empty project. Are you talking about what you can do on start.spring.io?

**Comment by CodingSpiderFox** [comment]
yes, I was just about to start a new little OSS project: a REST interface to send commands to my pioneer VSX-932 amplifier via tcp.

I'd like to publish it under Apache 2.0 license so it would have been useful to be able to add the LICENSE file with the license text to the zip file which can be downloaded from start.spring.io
